src/breakpoints/index.ts

```diff
@@ -130,13 +111,32 @@ export namespace Breakpoints {
       Signal.clearData(this);
     }
 
-    private _selectedType: SessionTypes;
+    setBreakpoints(id: string, breakpoints: IBreakpoint[]) {
+      this._breakpoints.set(id, breakpoints);
+      this.changed.emit(breakpoints);
+    }
+
+    getBreakpoints(id: string): IBreakpoint[] {
+      return this._breakpoints.get(id) || [];
+    }
+
+    restoreBreakpoints(breakpoints: Map<string, IBreakpoint[]>) {
+      this._breakpoints = breakpoints;
+      this._restored.emit();
+    }
+
+    /*private printMap() {
+      this._breakpoints.forEach((value, key, map) => {
+        console.log(key);
+        value.forEach((bp) => console.log(bp.line));
+      });
+    }*/
+
+    private _breakpoints = new Map<string, IBreakpoint[]>();
```

Comment:
Should we make an `interface` out of it?

Reply:
Maybe a type alias would be enough?

Reply:
Sounds good :+1: